Create project-portable-simd.toml

KodrAus opened the pull request:
Adds the Portable SIMD project group team.

Links

- https://github.com/rust-lang/rfcs/pull/2977
- https://github.com/rust-lang/libs-team/issues/4

r? @Mark-Simulacrum

Mark-Simulacrum commented:
> 
ERROR validation error: member calebzulawski is missing
ERROR validation error: member hsivonen is missing
ERROR validation error: person `calebzulawski` is member of team `inside-rust-reviewers` but doesn't exist
ERROR validation error: person `calebzulawski` is member of team `project-portable-simd` but doesn't exist
ERROR validation error: person `calebzulawski` is member of team `wg-leads` but doesn't exist
ERROR validation error: person `hsivonen` is member of team `inside-rust-reviewers` but doesn't exist
ERROR validation error: person `hsivonen` is member of team `project-portable-simd` but doesn't exist
ERROR validation error: person `hsivonen` is member of team `wg-leads` but doesn't exist
ERROR validation error: person `workingjubilee` is member of team `inside-rust-reviewers` but doesn't exist
ERROR validation error: person `workingjubilee` is member of team `project-portable-simd` but doesn't exist
ERROR validation error: person `workingjubilee` is member of team `wg-leads` but doesn't exist

Looks like CI is failing, you'll need to follow directions in https://github.com/rust-lang/team#adding-a-person-to-the-repository to fix it

KodrAus commented:
Ah thanks! I'll get on it

KodrAus commented:
Alrighty, I've updated the group. It turns out since the `leads` is a mailing list I've kept just the members with accessible emails there.

I also updated my own email.
